control/runner.go

```diff
@@ -286,7 +286,7 @@ func (r *runner) HandleGomitEvent(e gomit.Event) {
 			return
 		}
 	case *control_event.UnloadPluginEvent:
-		// On plugin unload,  find the key and pool info for the plugin being unloaded.
+		// On plugin unload, first find the key and pool info for the plugin being unloaded.
```

Comment:
Maybe add something to this comment about the reason why we are looking for that info - In order to move any subscriptions that we are able to move for this plugin.